plugins/processors/awsappsignals/internal/cardinalitycontrol/metrics_limiter.go

```diff
@@ -19,7 +19,8 @@ import (
 )
 
 const (
-	UnprocessedMetricValue = "AggregatedHighCardinalityMetrics"
+	UnprocessedServiceOperationValue = "AllOtherOperations"
```

Comment:
Are these attributes going to be dimensions? Or just attributes in the emf log?

Reply:
It will be a fixed metric dimension value to replace those high cardinality values detected by metric limiter.  

For example, If `Operation` dimension has a random user id as the value, we will override the value with it.
Operation="Post /api/user/123(random id)"
=>
Operation="AllOtherOperations"

Reply:
Gotcha, just confirming that this is a value of the dimensions and we do not have to make changes to the emf exporter.